builtInFunctions/esdtLocalBurn.go

```diff
@@ -79,6 +86,11 @@ func (e *esdtLocalBurn) ProcessBuiltinFunction(
 		return nil, err
 	}
 
+	if e.enableEpochsHandler.IsConsistentTokensValuesLengthCheckEnabled() {
+		if len(vmInput.Arguments[1]) > core.MaxLenForESDTIssueMint {
```

Comment:
can we rename the core.MaxLenForESDTIssueMint const to something else as now it is used in alot of other functions. Something like: core.MaxLenForESDTValue ?

Reply:
added a TODO for this